tests/v4l2decode.cpp

```diff
@@ -321,9 +323,9 @@ int main(int argc, char** argv)
     fd = YamiV4L2_Open("decoder", 0);
     ASSERT(fd!=-1);
 
+#if __ENABLE_V4L2_GLX__
     x11Display = XOpenDisplay(NULL);
```

Comment:
On Apr 9, 2015 7:09 PM, "Xu Guangxin" <notifications@github.com> wrote:
>
> In tests/v4l2decode.cpp:
>
> > @@ -321,9 +323,9 @@ int main(int argc, char** argv)
> >      fd = YamiV4L2_Open("decoder", 0);
> >      ASSERT(fd!=-1);
> >
> > +#if __ENABLE_V4L2_GLX__
> >      x11Display = XOpenDisplay(NULL);
>
> but eglGetDisplay will failed, right? glGenTextures all dependent on
eglDisplay, we'd better get eglDisplay from zone and eglInitialize it.
>

It is not failing as NATIVE_DISPLAY_AUTO is in use and eglGetDisplay will
create a Libyami drmfd. This is specific to the test application and
libyami.

> --
> Reply to this email directly or view it on GitHub.